fs/glob_test.ts

```diff
@@ -138,4 +138,120 @@ testWalk(
   }
 );
 
+test({
+  name: "isGlob: pattern to test",
```

Comment:
There should be test case testing that something is not glob `assert(!isGlob("foo.js"));`

Reply:
The test cases include both valid and invalid glob patterns,
I have used these [test cases](https://github.com/micromatch/is-glob/blob/master/test.js).